### ISSUE TITLE
Some changes

### ISSUE THREAD

**Issue by applibs** [hybridauth/hybridauth#1174]
1.
Sometimes it hapen to me that response is in bad format.
At Curl.php a add new line:
`$response = str_replace(array('( {', '} )'), array('({', '})'), $response);`
after:
`$response = curl_exec($curl);`

Otherwise bad response is not well converted into stdObject later.

2.
Please add into Hybridauth.php info about version please.
Something like this.
```
/**
    * Hybridauth version.
    *
    * @var string
    */
    public static $version = '3.3.0';
```

3.
In file Collection.php i changed constructor into:
```
public function __construct($data = null)
    {
        $this->collection = new \stdClass();

        if (is_object($data)) {
            $this->collection = $data;
        }
        if (is_array($data)) {
            $this->collection = (object) $data;
        }

    }
```
Otherwise sometimes I got error with retype of $data.

Thank you.

**Comment by ApacheEx** [comment]
Hello, can you tell us with which provider you face this issue? and also which endpoint you call?

I think it's too tricky and there should be better solution.

**Comment by applibs** [comment]
I dont remember yet because I used HA 3 long time ago when it was not at master. So I implemented changes because there was some problems.

**Comment by chrisgraham** [comment]
I think this issue should be closed, as it is probably actually #1007 in disguise. I doubt the str_replace did anything.

**Comment by ApacheEx** [comment]
yeah, true. We need to have real case to provide such changes.
